client/components/search-themes/index.tsx

```diff
@@ -45,6 +45,10 @@ const SearchThemes: React.FC< SearchThemesProps > = ( { query, onSearch } ) => {
 	};
 
 	const updateInput = ( updatedInput: string ) => {
+		// Clean up duplicate criteria
+		updatedInput = updatedInput.replace( /(\b(feature|column|subject):.*[^ ]\b)(?=.*\1)/gi, '' );
```

Comment:
![image](https://user-images.githubusercontent.com/1525580/203680770-8047a256-16eb-4479-a9ad-e426aad38966.png)

Reply:
LOL, I usually use this for regex check

![](https://user-images.githubusercontent.com/10071857/203681498-96644e16-aeea-41f2-8467-8621eed1323a.gif)